src/usr/local/www/firewall_rules_edit.php

```diff
@@ -42,8 +42,8 @@
 }
 
 function is_posnumericint($arg) {
-	// Note that to be safe we do not allow any leading zero - "01", "007"
-	return (is_numericint($arg) && $arg[0] != '0' && $arg > 0);
+	// Integer > 0? (Note that to be safe we do not allow any leading zero - "01", "007")
+	return (is_numericint($arg) && $arg[0] != '0');
```

Comment:
The function is to return true only for positive numeric integer that does not have any leading zeros.
What if $arg is negative?